packages/manager/modules/enterprise-cloud-database/src/create/choose-cluster-config/choose-cluster-config.controller.js

```diff
@@ -1,10 +1,21 @@
 export default class {
+  constructor($timeout) {
+    'ngInject';
```

Comment:
Elsewhere we use `/* @ngInject */` above constructor. We should keep consistency